users/models/custom_user.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from users.choices import GENDER_CHOICES, EDUCATION_CHOICES
+
+
+class CustomUser(AbstractUser):
+    """
+        date_of_birth, experience, gender and education are nullable
+        in case we will use oauth2 and cannot automatically take this data
+
+        user can change the data in profile page
+
+        Django forms and serializer do not allow you to leave data empty.
+        """
+    image = models.ImageField(upload_to='profile_pics',
+                              blank=True, null=True)
+    date_of_birth = models.DateField(null=True)
+    experience = models.IntegerField(null=True)
+    gender = models.CharField(choices=GENDER_CHOICES, max_length=20, null=True)
+    education = models.CharField(choices=EDUCATION_CHOICES, max_length=50,
+                                 null=True)
+    email = models.EmailField(blank=True, max_length=254,
+                              verbose_name='email address', unique=True)
+
+    def save(self, **kwargs):
+        """
+        if img is too big we decrease img
+        because the less image is the less memory it takes
+        """
+        super(CustomUser, self).save(**kwargs)
+        if self.image:
```

Comment:
Seems like thumbnails are optional. Thus it would be nice to handle exceptions here in order to prevent crashing of app in case of any "surprises".